Common/UnitDefinitions/Molarity.json

```diff
@@ -30,7 +30,20 @@
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "mol/L" ]
+          "Abbreviations": [ "mol/L"]
+        }
+      ]
+    },
+    {
+      "SingularName": "Molar",
```

Comment:
Does `Molar` replace `MolesPerLiter`? If so, we should maybe add `"ObsoleteText": "Deprecated, use Molar instead.",` to line 27 and maybe merge the abbreviations - or should they both exist?

Reply:
No, they are [all equivalent](https://en.wikipedia.org/wiki/Molar_concentration#Units). I tried simply adding the abbreviations to the original MolesPerLiter unit, but failed to parse back anything except for "123 M" (like "123 mM").

Reply:
Ah, yes, this is currently not supported: https://github.com/angularsen/UnitsNet/issues/344

I really want to avoid adding a separate unit to work around this problem and rather fix the support for multiple abbreviations with prefixes to begin with. Would you be interested in taking a look at that? 

I don't have too much time to spend on this project these days, so it may take some time for me to get around to it.

Reply:
Ok, my latest commit removes the redundant Molar unit, adding it as abbreviation instead. I've annotated the failing test as Skip= Awaiting fix for #344 effectively reversing the dependency. I suggest we close this first, and then I'll have a ready test for the fix in #344 (which I think I have).

Reply:
Ok, that is fine. Let's try to close this big one first then.

Reply:
I just saw your last commit the proper usage of Theory + InlineData so started converting where appropriate.. I agree that it's much better this way- however, when done _properly_, tests become quite a bit more verbose- I'll commit a sample of two tests for you to review- then we can figure out what's the appropriate approach...